public/manifest.json

```diff
@@ -1,25 +1,25 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "What's Near Me",
+  "name": "What's Near Me",
   "icons": [
     {
-      "src": "favicon.ico",
+      "src": "map-marked-alt-solid.svg",
       "sizes": "64x64 32x32 24x24 16x16",
       "type": "image/x-icon"
     },
     {
-      "src": "logo192.png",
+      "src": "map-marked-alt-solid.png",
       "type": "image/png",
       "sizes": "192x192"
     },
     {
-      "src": "logo512.png",
+      "src": "map-marked-alt-solid.png",
```

Comment:
It might have slipped from my mind but do we know how many icon sizes we need? I’ve checked  for iOS, there’re 120, 152, 167, and 180 pixel sizes. For Android - 144 and 192 pixel sizes. For chrome app - 512px.
Our app has currently a 192px icon and a 512px icon. Do we need to add more sizes? It looks like , for example, we are missing custom app icons for iOS ?

Reply:
Wow 😮! It's new! .... I didn't need was necessary. TIL.

Reply:
I think this is a great question!

It looks like the code template we were given was covering the minimum with the two sizes, 192px and 512px. In the story for this issue, for who we are supporting, it is general and says:

"What platforms need to be supported and tested?

- iOS and Android"

Wikipedia does say the minimum is 192px and 512px as well.
https://en.wikipedia.org/wiki/Progressive_web_application#Installability_criteria

If the following is a decent resource, it says "While providing 192px and 512px icons will be sufficient in the vast majority of cases, you can provide additional icons to achieve pixel perfection." https://whatwebcando.today/articles/automate-pwa-app-icon-splash-screen-creation/

It would be easy to add more sizes to cover edge cases. The main reason I hesitate is in case it would unknowingly increase the workload down the line. 

Any further input would be awesome!

Reply:
If we are supporting iOS and Android, then it would probably be a good idea to have a few more sizes of map icons? 
For example, for iOS, this is what I've found in the Apple documentation:
Icon size:  iPhone - 180x180, 120x120 px; 
iPad Pro - 167x167 px; 
iPad, iPad mini --- 152x152 px;
Also, for iOS, it's important that there's no transparency in these icons.

Reply:
I like your thinking here, @jssckbl! The team will spend the last week or 2 branding the app, so I suggest we put a pin in this and come back to it once you all decide on the look and feel of the app. 192 and 512 are sufficient for our current needs. Great discussion, y’all!!!

Reply:
Thanks, @katia-utochkina!  I resized the map.png icon to 512px and square and made sure the background wasn't transparent.